annotations/anchor.md

```diff
@@ -0,0 +1,21 @@
+**_anchor_** (English); _anker_ (Danish); _Anker_ (German)
+
+_**akkeri** n., pl. akkerum_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o1281)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=2907)]  
+
+An object designed to prevent or slow the drift of a ship, by gripping the solid seabed under the body of water.  
+
+<div align="center">
+  
+  ![anchor from Ladby ship](../images/Anchor_Ladby.png)  
+  Anchor and Anchor Chain from Ladby ship (Nicolaysen Pl. V, Fig. 1)
+
+</div>
+Anchors were critical for controlling and docking Viking ships, especially in storms or raids. Anchors were used when camping aboard a ship at night or when preparing for stealthy attacks (Jesh 166). It is believed that viking anchors were made of iron or a composite of wood, stone, and rope. The former would have been expensive and used on large ships while the latter would be more affordable and used on small ships (Crumlin-Pederson, 146). 
+
+---
+
+  Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ NED-New edition. Woodbridge, Suffolk, UK ; Rochester, NY: 
```

Comment:
```suggestion
  Jesch, Judith. _Ships and Men in the Late Viking Age: The Vocabulary of Runic Inscriptions and Skaldic Verse._ (Boydell & Brewer, 2001). https://www.jstor.org/stable/10.7722/j.ctt163tb4f
```

Reply:
Moving to one line and removing place of publication, since no longer advised in Chicago 18